introduction/java_datatypes/Solution.java

```diff
@@ -0,0 +1,25 @@
+import java.util.Scanner;
+
+class Solution {
+ 
+  private static void dataType(long valueToCheckType) {
+    System.out.println(valueToCheckType + " can be fitted in:");
```

Comment:
Please only print inside main. Outside of it use StringBuilders to construct and return Strings which can be later printed in main.